tools/Deployment/deploy-tasks.ps1

```diff
@@ -88,7 +88,12 @@ function PublishToChocolatey {
     Push-Location $chocoHomeDir
     & $chocoCommand pack
     & $chocoCommand apiKey -k $token -source https://push.chocolatey.org/
-    & $chocoCommand push $nupkgName
+
+    $chocoLogFile = "$PSScriptRoot\choco-push.log"
+    & $chocoCommand push $nupkgName --log-file=$chocoLogFile
+    if (($LastExitCode -ne 0) -and (Test-Path $chocoLogFile)) {
+        Write-Host "choco push failed, get detailed errors from log:`r`n$(Get-Content $chocoLogFile -Raw -Encoding UTF8)"
```

Comment:
still print a message if `$LastExitCode -ne 0`?

Reply:
`choco push` writes summarized log which can contain error messages. I don't think we need print another message If there is no more error detail, could you please help double confirm if we still print a message if `$LastExitCode -ne 0`